lesson04/script.js

```diff
@@ -0,0 +1,19 @@
+const ul = document.getElementById('js-list');
+const listContents = [
+ {to: "bookmark.html", img: "1.png", alt:"画像1", text: "ブックマーク"},
+ {to: "message.html", img: "2.png", alt:"画像2", text: "メッセージ"}
+]
+
+const fragment = document.createDocumentFragment();
+for(const list of listContents){
```

Comment:
[Good(Just a comment)]
When I did this assignment, I used forEach, so I looked up the difference between it and for of.
It seems that for of has more advantages and I will try to use it in the future.
I learned a lot, thank you!
[https://web-begginer-log.com/js_for/](https://web-begginer-log.com/js_for/)

Reply:
I had never used 'for of' before, so I was surprised that it could be used like this!! This is very good! I learned a lot!:satisfied: